### ISSUE TITLE
Implement welcome message when bot joins

### ISSUE THREAD

**Issue by Panthr75** [Panthr75/little-devil#4]
**What platform are you specifically talking about?**
Bot

**What type of feature is it?**
Add

**What is your feature idea?**
The bot should send a basic embed about it's information whenever it joins a guild.

**Screenshots/Videos of prototype (optional)**
Possible code:
```js
client.on("guildCreate", (guild) => {

    ...
    let defaultChannel = guild.channels.get(guild.id);
    if (defaultChannel === undefined || !defaultChannel.permissionsFor(client.user).has("SEND_MESSAGES")) {
        defaultChannel = guild.channels.find(channel => channel.permissionsFor(client.user).has("SEND_MESSAGES"));
    }

    if (defaultChannel !== undefined) // send embed to channel
    ...

});
```

**Comment by Panthr75** [comment]
Seems like a good idea, but what about it sending a message in a non-related channnel?

**Comment by Panthr75** [comment]
Implemented in 0.20.0 alpha

**Comment by Panthr75** [comment]
Not setting as resolved as I haven't tested that little devil actually sends the embed in a guild it joins.
However, this is the current embed that it should send upon joining a guild
![image](https://user-images.githubusercontent.com/29237603/82602021-0b860480-9b76-11ea-8920-b6eb6e416af2.png)
